### ISSUE TITLE
Phoenix update hangs after clicking "Install and Relaunch"

### ISSUE THREAD

**Issue by alextegelid** [kasper/phoenix#261]
- Version: 2.6.2 (71)
- macOS: 10.15.5

The installer has been hanging every time for the past 4-5 updates or so. The "Update Phoenix" window won't respond so any action other than moving it around after clicking "Install and Relaunch". After a while the cursor changes to the spinning ball when hovering the window.

**Comment by kasper** [comment]
@alextegelid Terribly sorry to hear that! I personally haven’t noticed any issues. Do you see anything weird in system logs? Phoenix uses [Sparkle](https://sparkle-project.org) for updates, you might want to report this to them so that they can help to debug any potential issues with the framework. Have you tried reinstalling the app manually?

**Comment by alextegelid** [comment]
I haven't tried reinstalling the app. But I have some new information on the issue now since it hung up just now and I was able to check the system logs. 

Right after I clicked the "Install Update" button the window froze and I almost immediately opened the system logs. The process "MTLComplierService" reports crashes ever since and the installer seemed to be completely frozen. Then suddenly the "Ready to Install" window appeared but out of focus and seemingly without accepting any input (eg. clicking the "Install and Relaunch" button).

![Skärmavbild 2020-08-10 kl  20 50 56](https://user-images.githubusercontent.com/7058721/89819999-914d9680-db4c-11ea-9b9b-f5f8b0d67046.png)
_The list of errors goes on for many rows above if I scroll up_

The update went on for about 20 minutes. During which time the "Software Update" and "Ready to Install" windows appeared, froze and after a few minutes each disappeared again. Apparently the windows were taking user input but without the UI reflecting the clicks since the next window were shown.

After that I changed the `daemon: true` setting to `false` and tried to boot Phoenix again. It started and a was able to check for updates via the menu item. It seemed the update hadn't finished and the same update was prompted. This time I started the system logs before I clicked "Install Update" and (for the first time since the problems started) the installation ran through as fast and smooth as I would expect it to and without raising any errors. So weird, but it seems it's working now.

Hope this helps, and feel free to close this issue. I'll open a new one if the problem appears again.
Thanks for Phoenix!
